src/main/java/com/toofifty/easyblastfurnace/overlays/ItemStepOverlay.java

```diff
@@ -44,8 +54,24 @@ public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem widget
 
         if (step == null) return;
         if (!(step instanceof ItemStep)) return;
-        if (((ItemStep) step).getItemId() != itemId &&
-            ((ItemStep) step).getAlternateItemId() != itemId) return;
+
+        if (currentWidgetItem != null) {
```

Comment:
Maybe we could split items/overlays into two so we don't have to do any of these confusing checks (took me a bit to understand what's happening). How about `ItemStep`, `BankItemStep`, `ItemStepOverlay` and `BankItemStepOverlay`?

Reply:
I can't believe I didn't think to do this, hah. Thanks a lot for the idea, it's massively simplified the code, and fixed the sluggish item overlay when scrolling in the bank.